js/src/tooltip.js

```diff
@@ -636,6 +620,20 @@ const Tooltip = (($) => {
       return config
     }
 
+    _cleanTipClass() {
+      const $tip = $(this.getTipElement())
+      const tabClass = $tip.attr('class').match(BSCLS_PREFIX_REGEX)
+      if (tabClass !== null && tabClass.length > 0) {
+        $tip.removeClass(tabClass.join(''))
+      }
+    }
+
+    _handlePopperPlacementChange(data) {
+      if (data.originalPlacement !== data.placement) {
```

Comment:
@wojtask9 This change work on Popover too because Popover extend from Tooltip.

And the `if` shouldn't be removed because on `hide` for Popover and Tooltip we destroyed Popover instance